src/shared/main/cpp/native_glue.hh

```diff
@@ -64,7 +64,7 @@ namespace litecore {
         public:
             // Warning: If `critical` is true, you cannot make any further JNI calls (except other
             // critical accesses) until this object goes out of scope or is deleted.
-            jbyteArraySlice(JNIEnv *env, jbyteArray jbytes, bool critical = false);
+            jbyteArraySlice(JNIEnv *env, jbyteArray jbytes, size_t length = 0, bool critical = false);
```

Comment:
Looks like in all practicality you've eliminated the need for these default argument values.  Since a value is always passed into them.

Reply:
Yes.  Done